htdocs/includes/odtphp/odf.php

```diff
@@ -156,7 +157,21 @@ public function setVars($key, $value, $encode = true, $charset = 'ISO-8859')
      */
 	public function convertVarToOdf($value, $encode = true, $charset = 'ISO-8859')
 	{
-		$value = $encode ? htmlspecialchars($value) : $value;
+		if ($encode) {
```

Comment:
It is not possible to understand if this change is good or not without samples in input and expected value to compare in output.
So can you introduce a phpunit test that validate that convertVarToOdf do the right thing ?
How can we reproduce the bug ?
Do you have example of value input and output that is wrong currently ? the source ticket is not clear and it seems each change to fix a trouble break another one :-(

Reply:
Thanks for the advice. Writing a test helped me fix my original PR.
I edited the PR message for clarifications.